src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/RecreateAppealPdfHandlerTest.java

```diff
@@ -124,13 +124,37 @@ public void shouldNotCallPdfService() throws CcdException {
         verify(sscsPdfService, never()).generateAndSendPdf(eq(caseDetails.getCaseData()), any(), eq(idamTokens), any());
     }
 
+    @Test
+    public void shouldCallPdfServiceWhenSscsDocumentIsNull() {
```

Comment:
Should this be `shouldNotCallPdfServiceWhenSscsDocumentIsNull`?

Reply:
No, it should be shouldCallPdfService, here if not any document that means we have create the new pdf so it will call pdfService to regenerate the pdf

Reply:
Ok, you have added a condition in the code `if (caseData.getSscsDocument() != null)` but only one test!  Should not update an existing test to make sure that the call still happens when there are SscsDocuments or are there other tests that satisfy that already?

Reply:
Existing tests are satisfied the conditions if (caseData.getSscsDocument() != null) as either getSscsDocument() is empty or getSscsDocument() contains documents.
I've to add new test to create the scenario for this condition